pkg/operator/ceph/controller/spec.go

```diff
@@ -686,8 +685,9 @@ func PodSecurityContext() *v1.SecurityContext {
 // PodSecurityContext detects if the pod needs privileges to run
 func CephSecurityContext() *v1.SecurityContext {
 	context := PodSecurityContext()
-	context.RunAsUser = pointer.Int64(CephUserID)
-	context.RunAsGroup = pointer.Int64(CephUserID)
+	CephUserID := int64(CephUserID)
```

Comment:
I guess we can remove this

Reply:
we can't remove it completly but we can remove the in64 conversation since below that line it requires pointer value